.github/workflows/publish.yml

```diff
@@ -34,24 +34,7 @@ jobs:
       id: checks
       uses: gradle/actions/setup-gradle@417ae3ccd767c252f5661f1ace9f835f9654f2b5 # tag=v3
       with:
-        arguments: check -Pjunit-tags=!slow -x jcstress
-
-  slowerChecks:
-    # similar limitations as in prepare, but we parallelize slower tests here
-    name: slowerChecks
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # tag=v4
-      - name: setup java
-        uses: actions/setup-java@99b8673ff64fbf99d8d325f52d9a5bdedb8483e9 # tag=v3
-        with:
-          distribution: 'temurin'
-          java-version: 8
-      - name: run slower tests
-        id: slowerTests
-        uses: gradle/actions/setup-gradle@417ae3ccd767c252f5661f1ace9f835f9654f2b5 # tag=v3
-        with:
-          arguments: reactor-core:test -Pjunit-tags=slow -Pjcstress.mode=sanity
```

Comment:
`-Pjcstress.mode=sanity` had no impact as `reactor-core:test` does not include `jcstress` tasks